tensorflow/contrib/rnn/kernels/BUILD

```diff
@@ -0,0 +1,19 @@
+# Libraries and Kernels for RNNs.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+load("//tensorflow:tensorflow.bzl", "tf_kernel_library")
+
+tf_kernel_library(
+    name = "lstm_ops",
+    prefix = "lstm",
+    deps = [
+        "//tensorflow/contrib/rnn:lstm_ops_op_lib",
+        "//tensorflow/core:framework",
```

Comment:
@keveman , i had to hack up the tf_custom_op_library a little bit (added hdrs field) and now it compiles and runs the test successfully, however, i feel like the solution isn't correct. pls see:

tensorflow/contrib/rnn/BUILD

i had to put in the kernels/lstm_ops.cc into the gpu_srcs field, this seems ackward?

shouldnt we refactor out the:

tf_custom_op_library

into something like

tf_custom_op_library AND tf_custom_kernel_library

then we can more appropriately build the kernel first then the op?